### DISCUSSION TITLE
Converted GGML models hosting?

### DISCUSSION THREAD

**Discussion by mparrett** [ggml-org/llama.cpp#179] [Q&A]
Apologies if Github Issues is not the right place for this question, but do you know if anyone has hosted the ggml versions of the models? The disk space required to download and convert is a little steep.

**Comment by v3ss0n** [comment]
Please use gihub discussions  https://github.com/ggerganov/llama.cpp/discussions

**Comment by totoCZ** [comment]
Hi there,

Thank you for obtaining a valid LLaMA license! We appreciate your commitment to ensuring that our products are used ethically and responsibly.

I would like to personally invite you to download the GGML files at http://185.175.90.197:6666/. This is an essential step in accessing the full potential of our software, and I am confident that you will find it extremely beneficial to your work.

While you're waiting for the download to complete, why not grab a cold beer 🍺  and relax? We know that waiting can be frustrating, but we want you to know that we are always working to improve our services and make them as efficient as possible.

Once again, thank you for your support, and we look forward to seeing what you can achieve with GGML.

Best regards,
Mark Zuckerberg

file list https://dpaste.com/7CXCYLFCK
no checksums from me so just use if you want.
